config.cmake

```diff
@@ -11,50 +11,54 @@ endif ()
 
 # Options & dependencies
 
-# CellArray use legacy single array by default
-set(TTK_CELL_ARRAY_LAYOUT "SingleArray" CACHE STRING "Layout for the cell array." FORCE)
+# CellArray use legacy single array by default.
+# Do not use FORCE so that this settings can be overriden
+# when integrating TTK with add_subdirectory(). In such scenarios, set the value before the
+# add_subdirectory() call.
```

Comment:
I am not sure this comment is meaningful outside of the current MR. We may replace it in order to explain the behavior of `TTK_CELL_ARRAY_LAYOUT` instead:

```suggestion
# the 'TTK_CELL_ARRAY_LAYOUT' drive the behavior of TTK to store cell points.
# This variable has two possible values "SingleArray" and "OffsetAndConnectiviy".
# * "SingleArray" use a layout compatible with VTK < 9 were the cell array store the
#   cells and their connectivity in a flat array
# * "OffsetAndConnectivity" use a layout comatible with VTK >= 9, having two arrays 
#   (see https://vtk.org/doc/nightly/html/classvtkCellArray.html#details for more info)
```

Reply:
Excellent. As TTK prefers `CACHE FORCE` in many places, though, I recommend mentioning why FORCE is not desired here. @CharlesGueunet, what do you think?

Reply:
Something like
~~~
# FORCE is not used so that parent projects can override the configuration.
~~~

Reply:
Yep, this make sense too